### PR TITLE
Feature/waypoint scrolling

### PR DESCRIPTION
**Pull request by ilikescience**
@endtwist plz review/merge!

@Sidlav feel free to browse through if you're curious.

### CODE REVIEW

File: assets/scripts/index.js [hunk 1]
```diff
@@ -32,10 +34,10 @@ const setClockHands = (date) => {
     };
     if (moment(date).isValid()) {
         const countdown = moment().countdown(date);
-        r(document.getElementById('day'), 12 * countdown.days);
-        r(document.getElementById('hour'), 30 * (countdown.hours % 12) + countdown.minutes / 2);
-        r(document.getElementById('min'), 6 * countdown.minutes);
-        r(document.getElementById('sec'), 6 * countdown.seconds);
+        r(document.querySelectorAll('#day')[0], 12 * countdown.days);
```
Comment (endtwist) [inline assets/scripts/index.js:37]: Just an aside, `document.querySelector('#day')` would grab the singular item, and wouldn't require `[0]` then.

File: assets/scripts/index.js [hunk 2]
```diff
@@ -47,24 +49,70 @@ const setCopy = (text) => {
...
-        eventCopy = `The next #crappyhour <a href="#">can't come soon enough</a>.`;
+        eventCopy = `The next #CrappyHourNYC <a href="#">can't come soon enough</a>.`;
     }
 
     setClockNumbers(clockVal);
     setClockHands(clockVal);
     setCopy(eventCopy);
+};
+
+setInterval(function() {
```
Comment (endtwist) [inline assets/scripts/index.js]: `setInterval(setClocks, 1000);`

File: assets/scripts/index.js [hunk 3]
```diff
@@ -47,24 +49,70 @@ const setCopy = (text) => {
...
+                updateContentHeight();
+            }
+        }
+    });
+};
+
+onScrollInit();
+
+$(document).ready(function() {
+    console.log('hi');
```
Comment (endtwist) [inline assets/scripts/index.js]: Oops?

File: gulp/serve.js [hunk 4]
```diff
@@ -28,13 +28,13 @@ module.exports = function(gulp) {
...
             // From https://github.com/BrowserSync/browser-sync/issues/823
             // due to https://github.com/localtunnel/localtunnel/issues/81
-            bs.emitter.on('service:running', function() {
-                bs.instance.tunnel.tunnel_cluster.on('error', function(err) {
-                    if(err.toString().indexOf('firewall') === -1)
-                        throw err;
-                    console.log('localtunnel connection lost; reconnecting...');
-                });
-            });
+            // bs.emitter.on('service:running', function() {
```
Comment (endtwist) [inline gulp/serve.js]: Can we just delete this block entirely?

**Comment by ilikescience** [comment]
@endtwist think I got most of your comments.

**Pull request merged**